src/Appwrite/Utopia/Response/Model/AttributeEnum.php

```diff
@@ -28,7 +28,7 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Array of elements in enumerated type.',
                 'default' => null,
-                'example' => 'element',
+                'example' => "['elementA','elementB']",
```

Comment:
Are you sure this is the right way to give an example of an array? I was sure that on the docs we just build the array from the single value example, but I see a lot of inconsistency, let's double check this and make sure we are consistent about the type here.

Reply:
Yeah I wasn't sure either 😑 Let me build a preview and check.

Reply:
Yep this is wrong. I need to fix these and check them manually.

Reply:
<img width="801" alt="Screen Shot 2022-08-11 at 2 19 10 PM" src="https://user-images.githubusercontent.com/29069505/184211026-8b876c80-e15c-4227-bf61-3339d842d511.png">
I found the proper syntax :)